packages/client/tests/functional/_utils/setupTestSuiteEnv.ts

```diff
@@ -208,7 +208,16 @@ export function setupTestSuiteDbURI(suiteConfig: Record<string, string>, clientM
       return { envVarName, newURI }
     })
 
-  const databaseUrl = newURI.replace(DB_NAME_VAR, dbId)
+  let databaseUrl = newURI
+  // Vitess takes about 1 minute to create a database the first time
+  // So we can reuse the same database for all tests
```

Comment:
Why do we usually _not_ do that?

Reply:
Because we need full isolation for tests usually.
For Vitess we can break isolation without problem now but when we run more tests on vitess we would need to adapt this, maybe we could pass `options?.databaseName,` and if provided use that instead of a cuid.
Which would make it flexible to control isolation where needed.

Reply:
Note that there are several jest workers running in parallel.  So, if one of the workers resets the and the other one currently runs tests against it, we might have a problem. We might either: 
- ensure each workers talk to it's own DB. I think easiest way to do so would be to include `pid` in db name.
- limit worker count to 1 - but that might negate all of performance gains

Reply:
We only have tests for vitess in `relationMode` directory and they run separately from other functional tests.
We should change this later if we add vitess tests to the functional tests probably.
But for now we have a clear gain of at least 10-15min where the relationMode=prisma tests would be the last action running.
So we win 10-15 waiting time for each PR